libraries/AP_HAL/board/linux.h

```diff
@@ -301,6 +301,8 @@
     // Define Notify
     #define OBAL_NOTIFY_LED
     
+    //Enable PPM - uncomment below line
+    //#define OBAL_RCINPUT_PPM
```

Comment:
why do we require people to uncomment this line and rebuild?

Reply:
do you prefer to define it as another board.. maybe OBAL-PPM with same configuration and this option is active?

Reply:
Why do you need it to be another board? What is the incompatibility?

Reply:
I need to enable ppl. to compile for pwm & ppm.
so I can think of two option...either editing the code. or define
make obal
make obal_ppm
kindly advise if there is a better approach to implement it.